app/utils/CurrentDateTime.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import javax.inject.{Inject, Singleton}
+import org.joda.time.{DateTime, DateTimeZone}
+
+@Singleton
+class CurrentDateTime @Inject()() {
+
+  def getDateTime: DateTime = DateTime.now(DateTimeZone.UTC)
+}
+
```

Comment:
Since the implementation of this was removed in the submit return controller, this file should be removed.